index.js

```diff
@@ -28,7 +28,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+
+  mickeyMouse = [];
```

Comment:
I like it!  But in a professional setting we'd want a much more descriptive variable name.